bisand/tibber-api
# Add support for sendpushnotification

**Why**
After using the tibber-api for a private project I wanted to send a push notification to my TIbber app as part. Since the tibber-api package was missing this I decided to implement support for it.


**How to test:** 
- Change the tibber apikey to your own in the /examples/push-notification.ts and run ts-node push-notification.ts
- The added test 'TibberQuery.sendPushNotification() should return an error when using demo user' is creating a call to Tibber using the demo token, so the test is asserting that the error received is 'operation not allowed for demo user'
